src/Bridges/Constraint/Constraint.jl

```diff
@@ -22,97 +22,115 @@ for filename in readdir(joinpath(@__DIR__, "bridges"); join = true)
 end
 
 """
-    add_all_bridges(bridged_model, ::Type{T}) where {T}
+    add_all_bridges(model, ::Type{T}) where {T}
 
-Add all bridges defined in the `Bridges.Constraint` submodule to
-`bridged_model`. The coefficient type used is `T`.
+Add all bridges defined in the `Bridges.Constraint` submodule to `model`.
+
+The coefficient type used is `T`.
 """
-function add_all_bridges(bridged_model, ::Type{T}) where {T}
```

Comment:
Can you rename `bridged_model` to `model` in a separate PR ? Keeping a clean diff for this PR is important

Reply:
Will do in https://github.com/jump-dev/MathOptInterface.jl/pull/2598